humblebundle.py

```diff
@@ -728,10 +736,15 @@ def main(argv=None):
         hb.update()
 
     if args.list is not None:
+        if args.platform is not None:
+            keys = filterGames(hb.games,args.platform)
```

Comment:
> I believe the whole filterGames() function can be replaced by a one-line, such as:
`keys = [keys for keys in games if any(_ for _ in games[keys]['downloads'] if _['platform'] == args.platform)]`

It could yes, but IMHO that would make the code less readable (at least to my non-python trained eyes).

```python
keys = [keys for keys in games for downloads in games[keys]['downloads'] if platform in downloads['platform']]
```

This would also work and doesn't use `_` or `any()`.